pkgs/development/tools/tora/default.nix

```diff
@@ -6,13 +6,13 @@ let
 
 in mkDerivation rec {
   name = "tora-${version}";
-  version = "3.1";
+  version = "unstable-2019-04-10";
```

Comment:
There is a release it's just not tagged https://github.com/tora-tool/tora/commit/39bf2837779bf458fc72a9f0e49271152e57829f